elfeed-org.el

```diff
@@ -85,20 +85,21 @@ Return t if it does or nil if it does not."
 
 (defun rmh-elfeed-org-mark-feed-ignore (url)
   "Set tag `rmh-elfeed-org-ignore-tag' to headlines containing the feed URL."
-  (dolist (org-file rmh-elfeed-org-files)
-    (with-temp-buffer
```

Comment:
`with-temp-buffer` made this function ineffective. The changes were not saved to the disk